blank_project/package.json

```diff
@@ -9,9 +9,9 @@
     "deploy:contract": "near deploy",
     "deploy:pages": "gh-pages -d dist/",
     "deploy": "npm run build && npm run deploy:contract && npm run deploy:pages",
-    "prestart": "npm run build:contract && npm run dev:deploy:contract",
-    "start": "CONTRACT_NAME=$(cat neardev/dev-account) parcel src/index.html",
-    "dev": "nodemon --watch assembly -e ts --exec 'npm run start'",
+    "prestart": "npm run build:contract && npm run dev:deploy:contract && node ./scripts/near-shell-check ",
```

Comment:
this doesn't make sense – as this template will only get used to create new projects, there is nothing to migrate

Reply:
removed